qa/L0_shared_memory/shared_memory_test.py

```diff
@@ -348,6 +370,28 @@ def test_infer_offset_out_of_bound(self):
         self.assertIn("Invalid offset for shared memory region", error_msg[0])
         self._cleanup_server(shm_handles)
 
+    def test_infer_byte_size_out_of_bound(self):
```

Comment:
Could you please test it for CUDA shared memory as well?

Reply:
We have a ticket for GPU shared memory. Can we add that test in that PR?

Reply:
Sounds good.